### ISSUE TITLE
Funnel values are too small for visualization

### ISSUE THREAD

**Issue by acanto95** [keen/keen-dataviz.js#178]
The last values on my funnel are too small and it makes the representation impossible to read.

Is there any adjustments I can make so that those values can be legible?

Thanks!

![image](https://user-images.githubusercontent.com/22454328/57098819-95033400-6ce0-11e9-908c-c2749204d30d.png)

**Comment by dariuszlacheta** [comment]
Hello @acanto95 

You can customize values in css. In Your case main values have a class called `.text-main` and percentages have `.text-second`. To change the color You need to use `fill` atribute because it is svg element, size You can adjust with `font-size`.

Here You can have jsfiddle example of how this can be done:
https://jsfiddle.net/dariuszlacheta/zhb7kcnp/2/

**Comment by adamkasprowicz** [comment]
Hi @acanto95 ,

We've just published a fix for this issue - keen-dataviz@3.10.1
Please use the `minimalSize` property to set the minimum width of the step [in pixels].

Sample
```javascript
new KeenDataviz({
          container: '.chart-demo-funnel',
          type: 'funnel',
          funnel: {
            minimalSize: 200,
          },
          results
        })
```

**Comment by acanto95** [comment]
Thank you @adamkasprowicz 

It worked fined, I just had to upgrade to the latest version and add "minimalSize" .
